tests/src/test/scala/com/nvidia/spark/rapids/udf/URLEncode.scala

```diff
@@ -32,9 +32,9 @@ class URLEncode extends Function[String, String] with RapidsUDF with Serializabl
   override def apply(s: String): String = {
     Option(s).map { s =>
       URLEncoder.encode(s, "utf-8")
-          .replace("+", "%20")
-          .replace("*", "%2A")
-          .replace("%7E", "~")
+        .replace("+", "%20")
+        .replace("*", "%2A")
+        .replace("%7E", "~")
```

Comment:
Scala style is [4 spaces on continuation](https://github.com/NVIDIA/spark-rapids/blob/branch-22.02/docs/dev/idea-code-style-settings.xml#L35), so this should not have been changed.
```suggestion
          .replace("+", "%20")
          .replace("*", "%2A")
          .replace("%7E", "~")
```